CHANGELOG.md

```diff
@@ -6,10 +6,24 @@ and this project adheres to
 
 ## [Unreleased]
 
-### Fixed
+### Added
+
+- cosmwasm-std: Implement `Mul` and `MulAssign` for `Uint128`.
+- cosmwasm-std: Implement `FromStr` for `Uint128`, `Uint256`, and `Uint512`.`
+
+### Changed
 
 - cosmwasm-std: Implement `Decimal::from_ratio` using full uint128
   multiplication to support a wider range of input values.
```

Comment:
I had this in the Fixed section before. However, I don't have strong opinions where it belongs.